homeassistant/components/prusalink/__init__.py

```diff
@@ -88,6 +88,8 @@ async def async_migrate_entry(hass: HomeAssistant, config_entry: ConfigEntry) ->
             )
             try:
                 await api.get_info()
+            except aiohttp.client_exceptions.ClientConnectionError as ex:
+                raise ConfigEntryNotReady(f"Error connecting to device: {str(ex)}" from ex
```

Comment:
We [don't currently support](https://github.com/home-assistant/core/blob/dev/homeassistant/config_entries.py#L527-L530) `ConfigEntryNotReady` being raised from the migration code. Probably should.

Reply:
Ahh, right - doesn't look too tricky to add the same retry logic to the migration step :thinking: If so how would you suggest progressing - maybe make a separate PR for that, park this one until that is done?